ci/jenkins/test.sh

```diff
@@ -353,6 +353,8 @@ function deliver_antrea_windows {
             sleep 5
             ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "W32tm /resync /force" | grep successfully && break
         done
+        # Clean netnat rules
```

Comment:
I feel it is better to run such cleanup commands in the stage to cleanup the setup but not in startup stage.